### PR TITLE
feat(tutorial): complete step 1

### PR DESCRIPTION
**Pull request by vkpro**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/components/TutorialHeader/TutorialHeader.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { Link } from 'react-router-dom';
+
+const TutorialHeader = () => (
+  <Header aria-label="Carbon Tutorial">
+    <SkipToContent />
+
+    <HeaderName element={Link} to="/" prefix="IBM">
+      Carbon Tutorial
+    </HeaderName>
+
```
Comment (kellychurchill) [inline src/components/TutorialHeader/TutorialHeader.js]: Youre missing the `HeaderNavigation` element. See https://www.carbondesignsystem.com/tutorial/react/step-1/#import-and-export-the-header

File: src/index.scss [hunk 2]
```diff
@@ -5,6 +5,12 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 
 @link https://github.com/carbon-design-system/carbon/issues/2789
 */
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+$feature-flags: (
```
Comment (kellychurchill) [inline src/index.scss]: This should come before the styles import. See https://www.carbondesignsystem.com/tutorial/react/step-1/#add-ui-shell-sass

File: src/components/TutorialHeader/TutorialHeader.js (hunk 1 above)
Comment (vkpro) [reply src/components/TutorialHeader/TutorialHeader.js]: @kellychurchill thank you for review. Fixed.

File: src/index.scss (hunk 2 above)
Comment (vkpro) [reply src/index.scss]: @kellychurchill Fixed.

**Pull request closed without merging**